gradle/libs.versions.toml

```diff
@@ -1,32 +1,33 @@
 [versions]
 # Kotlin
-kotlin = "1.9.10"
+kotlin = "1.9.22"
 kotlin-coroutines = "1.7.3"
+kotlinx-serialization = "1.6.2"
 dokka = "1.9.0"
 
 # Android
-android-build-gradle = "8.1.1"
+android-build-gradle = "8.2.1"
 
 # Androidx
-androidx-navigation = "2.7.3"
+androidx-navigation = "2.7.6"
 androidx-appcompat = "1.6.1"
 androidx-core = "1.12.0"
-androidx-test-core = "1.5.0"
-androidx-lifecycle = "2.6.2"
+androidx-lifecycle = "2.7.0"
 androidx-constraintlayout = "2.1.4"
-androidx-room = "2.5.2"
-androidx-annotation = "1.7.0"
+androidx-room = "2.6.1"
```

Comment:
Suggestion - Maybe the version bumps should be in a different PR?  Seems like they were all shotgun updated which would make it a little problematic to isolate a particular version update or set of updates that lead to any misbehavior.

Reply:
Many of these updates only affect the demo/test app for Harmony. The only new libraries included for Harmony are the update to the annotations, and the Kotlin Serialization library.